UI/Pages/TuitionPartner.cshtml

```diff
@@ -3,305 +3,392 @@
                 {
-                    <th scope="col" class="govuk-table__header">@subjectName</th>
+                  <th scope="col" class="govuk-table__header">@subjectName</th>
                 }
-            </tr>
+              </tr>
             </thead>
             <tbody class="govuk-table__body">
-            @for (var groupSize = 1; groupSize < 7; groupSize++)
-            {
+              @for (var groupSize = 1; groupSize < 7; groupSize++)
+              {
                 <tr class="govuk-table__row">
-                    <th scope="row" class="govuk-table__header">@(((Domain.Enums.GroupSize)groupSize).DisplayName())</th>
-                    @foreach (var (_, prices) in subjects)
+                  <th scope="row" class="govuk-table__header">@(((Domain.Enums.GroupSize)groupSize).DisplayName())</th>
+                  @foreach (var (_, prices) in subjects)
+                  {
+                    if (prices.TryGetValue(groupSize, out var price))
                     {
-                        if (prices.TryGetValue(groupSize, out var price))
-                        {
-                            <td class="govuk-table__cell">@(Model.Data.IsVatCharged ? price.AddVAT().FormatPrice() : price.FormatPrice())</td>
-                        }
-                        else
-                        {
-                            <td class="govuk-table__cell"></td>
-                        }
+                      <td class="govuk-table__cell">@(price.FormatPrice())</td>
                     }
+                    else
+                    {
+                      <td class="govuk-table__cell"></td>
+                    }
+                  }
                 </tr>
-            }
+              }
             </tbody>
-        </table>
-    }
-}
+          </table>
+
+          @if (@Model.Data.IsVatCharged)
+          {
+            <h3 class="govuk-heading-s">Tuition prices (including VAT)</h3>
+            <table class="govuk-table" data-testid="full-pricing-table-@($"{tuitionType.DisplayName()} {keyStage.DisplayName()}".ToSeoUrl())">
+              <caption class="govuk-table__caption govuk-table__caption--m">@tuitionType.DisplayName() - @keyStage.DisplayName()</caption>
+              <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                  <th scope="col" class="govuk-table__header">Tuition group size</th>
+                  @foreach (var (subjectName, _) in subjects)
+                  {
+                    <th scope="col" class="govuk-table__header">@subjectName</th>
+                  }
+                </tr>
+              </thead>
+              <tbody class="govuk-table__body">
+                @for (var groupSize = 1; groupSize < 7; groupSize++)
+                {
+                  <tr class="govuk-table__row">
+                    <th scope="row" class="govuk-table__header">@(((Domain.Enums.GroupSize)groupSize).DisplayName())</th>
```

Comment:
The `Domain.Enums` Qualifier is redundant for the `Domain.Enums.GroupSize.` We already have a using statement on that file for this.

Reply:
This was copied from existing code, but is a good spot, I've removed all the redundant `Domain.Enum.`